docs/user-guide/reference/join-vertex.md

```diff
@@ -0,0 +1,21 @@
+# Joins and Cycles
+
+As of Numaflow v0.10, Pipeline Edges can be defined so that multiple Vertices send to 1. This includes:
+- UDF Map Vertices
+- UDF Reduce Vertices
+- Sink Vertices
+
+Please see the following examples:
+- [Join on Map Vertex](https://github.com/numaproj/numaflow/blob/main/examples/11-join-on-map.yaml)
+- [Join on Reduce Vertex](https://github.com/numaproj/numaflow/blob/main/examples/11-join-on-reduce.yaml)
+- [Join on Sink Vertex](https://github.com/numaproj/numaflow/blob/main/examples/11-join-on-sink.yaml)
+
+## Cycles
+
+A special case of a "Join" is a **Cycle** (a Vertex which can send either to itself or to a previous Vertex.) An example use of this is a Map UDF which does some sort of reprocessing of data under certain conditions such as a transient error.
+
+Cycles are permitted, except in the case that there's a Reduce Vertex at or downstream of the cycle. (This is because a cycle inevitably produces late data, which would get dropped by the Reduce Vertex.)
```

Comment:
```suggestion
Cycles are permitted, except in the case that there's a Reduce Vertex at or downstream of the cycle. (This is because a cycle inevitably produces late data, which would get dropped by the Reduce Vertex.). For this reason, Cycles should be used sparingly.
```